### ISSUE TITLE
Fails to start on target code compiled-in kernel (no modules)

### ISSUE THREAD

**Issue by JuniorJPDJ** [open-iscsi/targetcli-fb#186]
My kernel has everything built-in, no modules support.
This is my kernel config: [config.txt](https://github.com/open-iscsi/targetcli-fb/files/6759306/config.txt)
I've enabled TCM, but targetcli fails to start:
```
# targetcli
[Errno 2] No such file or directory: '/sys/module/target_core_mod/holders'
# ls /sys/module 
auth_rpcgss  configfs    dm_bufio     dns_resolver      iscsi_tcp  libiscsi_tcp  nbd   nfsv4                    printk    rng_core              sg        sunrpc     workqueue
block        crc_t10dif  dm_mod       firmware_class    kernel     lockd         nfs   nfs_layout_flexfiles     random    scsi_mod              spurious  tcp_cubic
cifs         cryptomgr   dm_snapshot  iscsi_target_mod  libiscsi   loop          nfsd  nfs_layout_nfsv41_files  rcupdate  scsi_transport_iscsi  sr_mod    vhba
```
Am I missing something, or it's a bug?

**Comment by maurizio-lombardi** [comment]
Hello, what version of rtslib are you using? This should be fixed in v2.1.74

**Comment by JuniorJPDJ** [comment]
I just updated void-linux repositories to use 2.1.74.\
I've other problem now, but this one seems to disapper. Thanks ;D
